### ISSUE TITLE
unable to get user_attributes from userinfo endpoint

### ISSUE THREAD

**Issue by shawnhsy** [cloudfoundry/uaa#573]
With uaa 3.13.0 (build from master). 
created a test client like this. 
```
testclient
    scope: openid user_attributes
    resource_ids: none
    authorized_grant_types: refresh_token password client_credentials user_token
    autoapprove: 
    authorities: user_attributes
    name: testclient
    lastmodified: 1490515591290
```
and configured ldap to map some attribute.
this mapping is fine, test it with 
```
curl -d "username=ZX0001&password=123456&client_id=testclient&grant_type=password&scope=user_attributes+openid&response_type=token+id_token" -u "testclient:123456" http://192.168.110.15:8080/oauth/toke
```
and the returned id_token did contains my custom attributes.

id_token decoded like this:
```
	{
 alg: "RS256",
 kid: "key-id-1",
 typ: "JWT"
}.
{
 sub: "9e96b9f2-b38b-47d3-b0ec-6275e66f3d5e",
 previous_logon_time: 1490512387247,
 user_name: "ZX0001",
 origin: "ldap",
 amr: [
  "ext",
  "pwd"
 ],
 iss: "http://localhost:8080/uaa/oauth/token",
 user_attributes: {
  uid: [
   "ZX0001"
  ],
  branch: [
   "ZX"
  ]
 },
 client_id: "testclient",
 aud: [
  "testclient"
 ],
 zid: "uaa",
 grant_type: "password",
 user_id: "9e96b9f2-b38b-47d3-b0ec-6275e66f3d5e",
 azp: "testclient",
 scope: [
  "openid"
 ],
 auth_time: 1490512690,
 exp: 1490516290,
 iat: 1490512690,
 jti: "91ca2fd4457a4b6082328df3001adac1",
 email: "aa@bb.com",
 rev_sig: "ec08d5a1",
 cid: "testclient"
}.
[signature]
```

and access_token like this:
```
	{
 alg: "RS256",
 kid: "key-id-1",
 typ: "JWT"
}.
{
 jti: "fca58d2446504df285473ae12049a12c",
 sub: "9e96b9f2-b38b-47d3-b0ec-6275e66f3d5e",
 scope: [
  "openid",
  "user_attributes"
 ],
 client_id: "testcleint",
 cid: "testcleint",
 azp: "testcleint",
 grant_type: "password",
 user_id: "9e96b9f2-b38b-47d3-b0ec-6275e66f3d5e",
 origin: "ldap",
 user_name: "ZX0001",
 email: "aa@bb.com",
 auth_time: 1490519330,
 rev_sig: "d5fdbfea",
 iat: 1490519330,
 exp: 1490522930,
 iss: "http://localhost:8080/uaa/oauth/token",
 zid: "uaa",
 aud: [
  "openid",
  "testcleint"
 ]
}.
```
but the userinfo endpoint just returns:
```
{"user_id":"9e96b9f2-b38b-47d3-b0ec-6275e66f3d5e","sub":"9e96b9f2-b38b-47d3-b0ec-6275e66f3d5e","user_name":"ZX0001","given_name":null,"family_name":"hh","email":"aa@bb.com","phone_number":null,"previous_logon_time":1490515683469,"name":" hh"}
```
user_attributes is not included. what went wrong?

**Comment by shawnhsy** [comment]
Turns out that ldap user_attributes is not saved in uaa database. the default config storeCustomAttributes is set coded to be false. can't find a place to override this config. 
Setting ldap.storeCustomAttributes:true in uaa.yml doesn't do the trick.

**Comment by sreetummidi** [comment]
Being addressed with UAA 4.4.0 in this story: https://www.pivotaltracker.com/story/show/144966265
